app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
```

Comment:
Different import types should be separated by blank line
Also it is a common practice to list imports from same module separated by commas